rackawareness/README.rst

```diff
@@ -99,4 +109,36 @@ Configure and Deploy Confluent Platform
    
      kubectl describe kafka
 
-     
\ No newline at end of file
+============================
+Check Pod and Rack Placement
+============================
+
+With rack awareness enabled, broker pods will be scheduled so
+there is one broker per node, and at the Kafka layer, Kafka will
+make smart decisions about where to place data for maximum fault tolerance.
+For example, Kafka will avoid placing all replicas for a partition on brokers
+in the same zone.
+
+#. Check the Node labels for ``topology.kubernetes.io/zone`` again:
+
+   ::
+
+    kubectl get node \
+     -o=custom-columns=NODE:.metadata.name,ZONE:.metadata.labels."topology\.kubernetes\.io/zone" \
+       | sort -k2
+
+#. Check that the ``broker.rack`` property has been set inside broker 0.
+
+   ::
+    
+     kubectl exec -it kafka-0 -- \
+       grep 'broker.rack' confluentinc/etc/kafka/kafka.properties
```

Comment:
Missing a pipe?

Reply:
No, this is how kubectl exec works. From https://kubernetes.io/docs/tasks/debug-application-cluster/get-shell-running-container/:

> Note: The double dash (--) separates the arguments you want to pass to the command from the kubectl arguments.

This command worked as-is when I tested it